source/_integrations/sighthound.markdown

```diff
@@ -63,3 +63,47 @@ source:
       required: false
       type: string
 {% endconfiguration %}
+
+To verify the integration, check if new entities appear as `camera.sighthound` and `image_processing.sighthound_camera_my_cam`
+
+## Process an Image
+
+When you want to process an image, you have to call `image_processing.scan` service and listen to the sighthound.person_detected and/or sighthound.vehicle_detected events.
+
+A simple example is via 2 two automations:
+
+- The first automation is triggered, when a Unify G4 doorbell detects a motion. It calls `image_processing.scan` service to send the camera image to the sighthound server for processing.
```

Comment:
Probably dont need to mention the brand of doorbell, just say its motion triggered

Reply:
Done.